workshop/00-initialization.md

```diff
@@ -2,29 +2,90 @@
 
 You can do so by typing `lb` or `lb app` and this will invoke the generator.
 
-### What's the name of your application?
-
-![LoopBack Initialization - step 1 - what is the name your application?](assets/lb-step1.png)
-
 The first step is to name the application.
 
-### Enter name of the directory to contain the project:
+### What's the name of your application?
+```
+     _-----_
+    |       |    ╭──────────────────────────╮
+    |--(o)--|    │  Let's create a LoopBack │
+   `---------´   │       application!       │
+    ( _´U`_ )    ╰──────────────────────────╯
+    /___A___\   /
+     |  ~  |
+   __'.___.'__
+ ´   `  |° ´ Y `
+
+? What's the name of your application? marketplace
+```
 
-![LoopBack Initialization - step 2 - what directory to be used for application?](assets/lb-step2.png)
+### Enter name of the directory to contain the project:
+```
```

Comment:
THis is a matter of preference, but in my code snippets, I only included the Yeoman character in the first code block. Up to you.